src/api/c/array.cpp

```diff
@@ -391,3 +392,47 @@ INSTANTIATE(af_is_bool        , isBool        )
 INSTANTIATE(af_is_sparse      , isSparse      )
 
 #undef INSTANTIATE
+
+
+af_err af_get_scalar(double* real_val, double* imag_val, const af_array arr)
```

Comment:
This follows what we were doing for reductions. 

@9prady9 We should probably fix this to be `void *` since it is similar to `af_get_host_ptr`. Also only return a single `void *` instead of two outputs.